jenkins/test-image/Dockerfile

```diff
@@ -43,6 +43,13 @@ RUN apt-get -o Acquire::Check-Valid-Until=false update && apt-get install -y \
 	--no-install-recommends \
 	&& rm -rf /var/lib/apt/lists/*
 
+# Download and symlink etcd. We need this for verification and integration tests.
+RUN export ETCD_VERSION=v3.0.4; \
```

Comment:
the problem with using hack/install-etcd.sh (which also exists with the fix I have, and with the go version we use, etc) is that different releases of kubernetes use different versions of etcd, Go, etc.

right now we run tests on all release branches with the latest image, which will likely result in incorrect behavior for older branches.